scripts/generate-content/contentful.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+const contentful = require("contentful-management");
+const { generateEvent } = require("./generators");
+
+const MAX_EVENTS = 150;
+
+const generateEvents = (spaceId, accessToken, count) => {
+  const client = contentful.createClient({
+    accessToken
+  });
+
+  client
+    .getSpace(spaceId)
+    .then(async space => {
```

Comment:
Since we use `async`/`await` here already, I think could make the whole `generateEvents` function `async` and use a try-catch. Up to you, though 🙂

Reply:
Good point, done